src/components/Gesture.svelte

```diff
@@ -133,6 +133,7 @@
     // Once duration is over (1000ms default), stop recording
     setTimeout(() => {
       unsubscribe();
+      console.log("RECEIVED SAMPLES", get(settings).numSamples, newData.x.length);
```

Comment:
Debug console message that print how many samples were recorded and how many the model expected.